src/main/UniversitiesInformation.java

```diff
@@ -6,34 +6,83 @@
 import java.util.Scanner;
 
 public class UniversitiesInformation {
-
+	private static String[] names = new String[14];
+	private static double[] overallAverages = new double[14];
+	private static double[] cutoff = new double[14];
+	private static int[] tuition = new int[14];
+	private static int[] classSize = new int[14];
+	private static double[] longitude = new double[14];
+	private static double[] latitude = new double[14];
 	public static ArrayList<University> universities = new ArrayList<>(14);
 
 	// Reads information from Universities.txt, and creates University objects for
 	// each University.
 	// Stores each instance in an ArrayList.
 
 	public static void setUniversities() {
```

Comment:
Instead of having a static method and calling UniversitesInformation.setUni...(), you would probably want to move all this into the constructor (which is non-existent right now) for OOP practice. Also, there are still static fields here...